CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Support for MirrorMaker 1 has been removed
 * Added support to configure `dnsPolicy` and `dnsConfig` using the `template` sections.
 * Store Kafka node certificates in separate Secrets, one Secret per pod.
+* Support for Strimzi Metrics Reporter for Kafka has been added.
```

Comment:
```suggestion
* Added support for Strimzi Metrics Reporter to the Kafka component.
```

Reply:
Consistency is always appreciated ;)

Reply:
Of course. Silly of me to miss that. Fixed now.